### PR TITLE
BumpTechnologiesInc.BumpTop version 2.5.6268

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16379)

### CODE REVIEW

File: manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BumpTechnologiesInc.BumpTop
```
Comment (ItzLevvie) [inline manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: BumpTechnologies.BumpTop
```

File: manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: BumpTechnologiesInc.BumpTop
```
Comment (ItzLevvie) [inline manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.installer.yaml:3]:
```suggestion
PackageIdentifier: BumpTechnologies.BumpTop
```

File: manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: BumpTechnologiesInc.BumpTop
```
Comment (ItzLevvie) [inline manifests/b/BumpTechnologiesInc/BumpTop/2.5.6268/BumpTechnologiesInc.BumpTop.yaml:3]:
```suggestion
PackageIdentifier: BumpTechnologies.BumpTop
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything else looks good to me:
image

**Comment by sanikdah** [comment]
What.....

**Comment by sanikdah** [comment]
How to fix the conflicting files....

**Comment by ItzLevvie** [comment]
You can close the PR; this manifest was added by https://github.com/microsoft/winget-pkgs/pull/16402.

**Comment by sanikdah** [comment]
oh, ok then.

**Pull request closed without merging**